plugins/modules/mysql_user.py

```diff
@@ -165,6 +164,13 @@
         fields names in privileges.
     type: bool
     version_added: '3.8.0'
+  attributes:
+    description:
+      - Create, update, or delete user attributes (arbitrary "key: value" comments) for the user.
+      - MySQL server must support the INFORMATION_SCHEMA.USER_ATTRIBUTES table. Provided since MySQL 8.0.
+      - To delete an existing attribute, set its value to False.
+    type: dict
+    version_added: '3.9.0'
```

Comment:
I did not bump the version in `galaxy.yml` yet; unsure what the process is for shipping new versions.